packages/Input/src/Input.js

```diff
@@ -117,6 +117,9 @@ const Input = props => {
     <div css={inputStyles} {...styleProps} className={rootClasses}>
       {renderIcon()}
       <input
+        aria-invalid={hasError}
+        aria-disabled={isDisabled}
+        aria-readonly={isReadOnly}
```

Comment:
I believe we should be setting these attributes on our native input elements as well? @mikrotron

Reply:
I think `disabled` and `readOnly ` are equal to aria-disabled and aria-readonly, but not 100% sure 😄

Reply:
I find some people mention `aria-disabled` is not needed if using `disabled` prop but don't find my opinion on the other attributes

Reply:
Yeah, if it's a native html element, `disabled` and `readonly` are equivalent.  It doesn't _hurt_ have have the extra `aria` attributes, but the redundancy is not necessary.

Reply:
@mikrotron opinion on `aria-invalid`?

Reply:
That one's good.  There's no native html equivalent.